### DISCUSSION TITLE
How about making two note types in Rnote?

### DISCUSSION THREAD

**Discussion by mahdihasan2006** [flxzt/rnote#1182] [General]
For example, Rnote will let users to choose from the following two note types while creating one:

1. **One page with infinite size**
2. **Multiple pages with specific size**
   Specific features to this type which aren't required in the previous one:
	- Adding or remove pages from any part of the document
	- Reordering pages
	- Page numbers for easy navigation

**Comment by Doublonmousse** [comment]
Part of this functionality is already there. I don't think we should remove options (keep infinite, semi infinite and continuous layout, and keep the fixed size one for adding page-related features).

Choosing page types when creating a new document is issue #1160 (template window upon creating).

For the fixed size mode, it's #639.

One particularity/weird thing is that even the fixed page size is an infinite layout. I think it does come with advantages (you can copy/paste and move elements between pages without issues) but also some disadvantages (you can write across pages border).

This means we probably have to have an edge case if that occurs whilst doing some page management (either stop the operation or split the stroke ?)
